crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -1,10 +1,16 @@
 # Nushell Environment Config File
 
 def create_left_prompt [] {
+    let home = ($env | get -i HOME | into string)
+    let dir = ([
+        ($env.PWD | str substring 0..($home | str length) | str replace $home "~"),
```

Comment:
Once you change the above to allow for Windows paths like `C:\some\path` you'll need to fix the string replace to do something like `str replace $home "~" -s`. The `-s` makes it a string replace versus a regex replace. The regex replace will fail because it thinks `C:\some\path` has escapes.

Reply:
Done, thanks.